packages/vanilla/src/cells/EnumCell.tsx

```diff
@@ -50,8 +50,8 @@ export const EnumCell = (props: EnumCellProps & VanillaRendererProps) => {
           .concat(
             options.map(optionValue =>
               (
-                <option value={optionValue} label={optionValue} key={optionValue}>
-                  {optionValue}
+                <option value={optionValue.value} label={optionValue.label} key={optionValue.value}>
+                  {optionValue.label}
```

Comment:
Not really a part of this PR but is `{optionValue.label}` and `label={optionValue.label}` not redundant? Also can we rename `optionValue` to `option`?